### PR TITLE
docs: community page translated into Korean

### PR DESCRIPTION
**Pull request by Gaic4o**
## Background

We translated the Using with community page.

If you think there are aspects that need improvement, please let me know, and I would be happy to make the adjustments. Thank you!

I have requested a Korean translation review from one reviewer in the FSD Discord Korean community to ensure that my Korean writing is well done. I would appreciate it if you could merge it after the review is completed.

### CODE REVIEW

File: i18n/kr/docusaurus-plugin-content-docs/community/index.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import NavCard from "@site/src/shared/ui/nav-card/tmpl.mdx"
+import { StarOutlined, SearchOutlined, TeamOutlined, VerifiedOutlined } from "@ant-design/icons";
+
+<NavCard
+    title="커뮤니티 리소스와 추가 자료" 
+    description="훌륭한 FSD 관련 비디오, 기사, 패키지를 모아둔 큐레이션 목록"
+    to="https://github.com/feature-sliced/awesome"
+    Icon={StarOutlined}
+/>
```
Comment (movie42) [inline i18n/kr/docusaurus-plugin-content-docs/community/index.mdx:21]: Awesome이라는 레포지토리가 있는데 https://github.com/sindresorhus/awesome#readme

여기의 시리즈중 하나로 Awesome FSD가 있다는 것을 소개하고 있는 것으로 보입니다. 

Awesome을 고유 대명사로 보고 'Awesome 자료' 또는 'Awesome 리소스'로 변경하는 것은 어떨지요? 

아래 description에 대해서는 이견 없습니다.

**Review by movie42 — APPROVED** [review]
번역해주셔서 감사합니다.

File: i18n/kr/docusaurus-plugin-content-docs/community/index.mdx (hunk 1 above)
Comment (Gaic4o) [reply i18n/kr/docusaurus-plugin-content-docs/community/index.mdx]: 좋은 거 같아요.. 감사합니다!

**Comment by movie42** [comment]
Hello Maintainer,

I have completed the review of this translation, and the contributor has incorporated the suggested changes. Upon re-review, I did not find any issues. Please take a look when you have time.

Thank you!

**Comment by Gaic4o** [comment]
@illright

**Review by illright — APPROVED** [review]

**Pull request merged**